events/artsy-learns-to-code-v2.md

```diff
@@ -0,0 +1,116 @@
+- An open mind
+- (Optional) Come with an idea of what they want to learn
+
+## Expectations of Mentoring Engineers
+
+- Mentors should expect to spend 1-2 hrs a week of their time to meet regularly and answer questions async on slack
+- Mentors should feel empowered to come up with their own teaching plan (and ideally share it with the rest of the
+  mentors!). Note: mentors should consider the extra time required to craft their own teaching plan.
+- Act as a resource for when learners have questions
+- Act as a motivator for when learners get frustrated
+  - Normalize frustration (https://www.youtube.com/watch?v=QYjPBcqo1-s)
+- If mentor group agrees to go the route of learning Processing.js, then the mentor should try to familiarize
+  themselves with p5js, or at least follow along with the course.
+
+## Teaching resources
+
+Here are a couple examples that [@sepans][sepans] have created:
+
+| Project Name + Concept it teaches                | Link                                              |
+| ------------------------------------------------ | ------------------------------------------------- |
+| Bouncy bubble (conditionals)                     | https://editor.p5js.org/sepans/sketches/8I6gA3O-1 |
+| Damien Hirst (loops)                             | https://editor.p5js.org/sepans/sketches/o34yoVD3y |
+| Damien Hist 2 (array to store randomized colors) | https://editor.p5js.org/sepans/sketches/RBjEojE3g |
+| Bouncy DVD with Object (objects)                 | https://editor.p5js.org/sepans/sketches/jmu-Gb0u7 |
+| 300 DVDs (objects + functions + arrays)          | https://editor.p5js.org/sepans/sketches/yaafYKYN7 |
+| Sunglass (capturing data and displaying it)      | https://editor.p5js.org/sepans/sketches/s9tmMft3R |
+| Load artwork image from artsy (API)              | https://editor.p5js.org/sepans/sketches/NbuDfgC47 |
+
+Here are tools that [@sepans][sepans] recommended to his group in the previous ALTC:
+
+| Tool Name                                      | Link                            |
+| ---------------------------------------------- | ------------------------------- |
+| Prettier: Format you code online               | https://prettier.io/playground/ |
+| Codepen: To write a small JS/HTML sketch       | https://codepen.io              |
+| VS Code: Code Editor                           | https://code.visualstudio.com/  |
+| Glitch: remix existing apps to create new ones | http://glitch.com               |
+
+Even more resources that [@sepans][sepans] compiled:
+
+| Resource Name                                            | Link                                          |
+| -------------------------------------------------------- | --------------------------------------------- |
+| Cool p5 sketches to tweak and play with                  | http://www.generative-gestaltung.de/2/        |
+| p5 DOM: use p5 to create and interact with HTML elements | https://p5js.org/reference/#/libraries/p5.dom |
```

Comment:
p5.dom is now merged into core, and is included in main reference page. no need to direct to a separate link :)